mlflow/server/js/src/components/MetricView.js

```diff
@@ -28,7 +28,7 @@ class MetricView extends Component {
             title={<span>{metricKey}</span>}
```

Comment:
If we conditionally include this title when there's a single runUuid, we avoid the issue @dbczumar found where it's confusing that the metric comparison page shows a single metric key as its title even when you're comparing multiple metrics.

Reply:
Even if there's a single runUuid, the metrics view can still be used to visualize multiple metrics for a single run. Therefore, I'm still in favor of "Metrics"

Reply:
Let's conditionally make it either `Metrics` or `{metric_name}` then.